### ISSUE TITLE
defaultEnvFile is not documented anywhere

### ISSUE THREAD

**Issue by henrikra** [react-native-config/react-native-config#212]
Looks like that I can set `defaultEnvFile` to my gradle file. But this is not documented anywhere. Could someone tell me an example and maybe update the docs also? :)

**Comment by drewandre** [comment]
Yeah I would love to see how this is used. I'm using `envConfigFiles` for multiple builds but the project is defaulting to `.env`, which doesn't exist in my project. I've tried adding it to my gradle.properties file but I can't seem to pull that into my build.gradle.

**Comment by kyle-ssg** [comment]
```
buildscript {
    ext {
        defaultEnvFile = file('.env').absolutePath.replace("/android", "")
        ...
```

For reference
